src/Aspire.Dashboard/Components/Pages/ConsoleLogs.razor.cs

```diff
@@ -261,6 +265,33 @@ private void UpdateMenuButton()
             Icon = new Icons.Regular.Size16.ArrowDownload()
         });
 
+        if (PageViewModel.HideLogDate)
+        {
+            _menuItems.Add(new()
+            {
+                OnClick = async () =>
+                {
+                    PageViewModel.HideLogDate = false;
+                    await this.AfterViewModelChangedAsync(_contentLayout, waitToApplyMobileChange: false);
+                },
+                Text = "Show log dates",
+                Icon = new Icons.Regular.Size16.Eye()
```

Comment:
Rather than change the icon by state, can we use a latched/checked icon state instead? Or maybe we just need a better icon. We use an eye icon for masking secrets throughout.

Reply:
Done:

![image](https://github.com/user-attachments/assets/b39dc547-40fd-4f93-91af-c82b49f1b2f5)